PDFTron/webviewer-ui
# Add Wildcard option to advanced search.

QA Steps:
- Launch WebViewer
- In the search panel, open it to see advanced search modes. 
- There should now be a Wildcard option
![image](https://user-images.githubusercontent.com/10825527/70086911-2b297080-15c8-11ea-8f99-84c059aad25a.png)

- Do a few wildcard searches to verify it works.
